get_item_lists.sh

```diff
@@ -1,55 +1,49 @@
-D=$(date +\%Y-\%m-\%d)
+#!/usr/bin/env bash
+
+set -euo pipefail
 
 # BOIA: 271744772
 bundle exec ruby get_ht_ids_by_auth.rb /l1/govdocs/authority_lists/txt/boia.auths.txt\
-  | sort -T tmpsort | uniq > data/boia_ht_item_ids.$D.txt 
-# comm -2 -3 data/boia_ht_item_ids.$D.txt data/boia_current_ht_item_ids.$D.txt\
-#  > data/boia_new_ht_item_ids.$D.txt
-# comm -1 -3 data/boia_ht_item_ids.$D.txt data/boia_current_ht_item_ids.$D.txt\
-#  > data/boia_deleted_ht_item_ids.$D.txt
- 
+    | sort -T tmpsort | uniq > data/boia_ht_item_ids.txt || {
```

Comment:
I am tempted to suggest breaking out common bash functions into a bash library file.

```
...
tmp_sort(){
    sort -T tmpsort | uniq
}
...
```